x-pack/plugins/siem/public/resolver/view/submenu.tsx

```diff
@@ -142,23 +150,35 @@ const NodeSubMenuComponents = React.memo(
      * When called with a set of `optionsWithActions`:
      * Render with a panel of options that appear when the menu host button is clicked
      */
+
+    const button = (
```

Comment:
:grey_question: may want to provide a less generic name here (like `popoverButton` or something) to prevent collisions / confusion